### ISSUE TITLE
change directory magic

### ISSUE THREAD

**Issue by RafalSkolasinski** [gopherdata/gophernotes#207]
Standard jupyter notebook (python) kernel allows me to use 
```bash
%cd ...
```
magic to change current directory.

It may be a nice feature to also have it for `gophernotes`.

**Comment by cosmos72** [comment]
Yes, it would be useful - especially now that we have the special commands
```
!some-shell-command [args]
```

**Comment by simongarisch** [comment]
Firstly, this is a nice project, so thanks for putting the effort into it.
As Rafal has mentioned, it'd be useful to enhance the golang notebooks with jupyter magic commands.
Using '%%javacript', for example, will work for the gophernotes docker python kernel but not for go.

**Comment by cosmos72** [comment]
I am implementing `%cd [path]` - it's almost trivial

But I have no idea how to run javascript from gophernotes - maybe there's a way to return a response that will be executed as javascript in the browser?

**Comment by simongarisch** [comment]
@cosmos72 No worries. I was more interested in this for leaning on JS plotting libraries. There may be some other ways I can approach this.
